forge/Dockerfile

```diff
@@ -0,0 +1,39 @@
+ARG PANGEO_BASE_IMAGE_TAG=master
+# Apache Beam requires the following to exist in docker images used to run beam
+# workloads:
+# 1. The apache-beam python package, with appropriate extras (for example, [gcp] if we
+#    we want to accss google cloud services)
+# 2. A 'beam boot' executable binary set as the entrypoint. This is a go program that
+#    actually launches the appropriate python code.
+#
+# These two need to be the same version to avoid any possible incompatibilites.
+#
+# In this image, we install the python package via pip (as the conda version seems currently
+# broken), and build the go executable by downloading the source & using a builder image
+# to build it. This ensures we meet the requirements needed for running beam while still
+# reusing most of the pre-existing infra we have for maintaining images.
+FROM golang:1.18-bullseye as beamboot
+
+# This version is specified twice in this Dockerfile, so if you change this one,
+# make sure to change the other one too!
+ARG BEAM_VERSION=2.40.0
+
+RUN curl -sSL https://github.com/apache/beam/archive/refs/tags/v${BEAM_VERSION}.tar.gz | tar -xzf -
+
+RUN cd beam-${BEAM_VERSION}/sdks/python/container && \
+    CGO_ENABLED=0 go build -o /beam-boot
```

Comment:
Why not pull this from the Beam Python SDK directly? Are you sure you need to compile the go sources again? I'm not sure this is necessary.

Reply:
@abkfenris yeah, we could pull the go binary alone out of the beam sdk image (assuming that is what you are referring to) and I think that should be ok too. My initial reading of the beam dockerfile assumed that the python package was also installed into `/opt/apache/beam` and I didn't want to pull *that* in. But I guess I can copy just the binary out?

Do you know if it's statically built or links to local libc?

Reply:
Knowing @abkfenris, he'll probably be interested in this conversation, but you probably meant to tag @alxmrs. 😄

Reply:
You mean I don't have to go down this rabbit hole right now? I mean I want to, but I really can't.

Reply:
@cisaacstern do you know what error you were getting?